diff --git a/node_modules/@floating-ui/react-dom-interactions/dist/floating-ui.react-dom-interactions.umd.js b/node_modules/@floating-ui/react-dom-interactions/dist/floating-ui.react-dom-interactions.umd.js
index ac0c919..df11519 100644
--- a/node_modules/@floating-ui/react-dom-interactions/dist/floating-ui.react-dom-interactions.umd.js
+++ b/node_modules/@floating-ui/react-dom-interactions/dist/floating-ui.react-dom-interactions.umd.js
@@ -181,7 +181,7 @@
       }
     });
 
-    if (useInsertionEffect) {
+    if (useInsertionEffect && false) {
       // eslint-disable-next-line react-hooks/rules-of-hooks
       useInsertionEffect(() => {
         ref.current = callback;
